lizard.py

```diff
@@ -674,20 +674,31 @@ def clang_warning_format(self):
 
 
 def print_warnings(option, scheme, warnings):
-    warning_count = 0
+    warning_count = None
     warning_nloc = 0
     warn_str = "!!!! Warnings ({0}) !!!!".format(
         ' or '.join("{0} > {1}".format(
             k, val) for k, val in option.thresholds.items()))
-    print("\n" + "=" * len(warn_str) + "\n" + warn_str)
-    print(scheme.function_info_head())
-    for warning in warnings:
-        warning_count += 1
+    for warning_count, warning in enumerate(warnings):
+        if warning_count == 0:
```

Comment:
@mdmedley I think here is the bug. The enumeration is zero-based, so if you have one warning, the warning count is 0. Classical off-by-one.